api/envoy/api/v3alpha/core/health_check.proto

```diff
@@ -70,14 +71,14 @@ message HealthCheck {
     }
   }
 
-  // [#next-free-field: 11]
+  // [#next-free-field: 13]
   message HttpHealthCheck {
     option (udpa.annotations.versioning).previous_message_type =
         "envoy.api.v2.core.HealthCheck.HttpHealthCheck";
 
-    reserved 7;
+    reserved 5, 7;
 
-    reserved "use_http2";
+    reserved "service_name", "use_http2";
```

Comment:
@mattklein123 I am not too such of this change, but this was recommended by the pre commit hooks.

Reply:
This is correct, due to deprecation.